### ISSUE TITLE
The aria-multiline attribute is invalid on combobox role

### ISSUE THREAD

**Issue by atarix83** [ng-bootstrap/ng-bootstrap#4181]
### Bug description:
When using the the typeahead component `aria-multiline="false"` is added to the input but is not allowed since is valid only for the `textbox` role while by using the typeahead component the role is `combobox`.
This is an accesibility issue because elements must only use allowed ARIA attributes.
  

### Link to minimally-working StackBlitz that reproduces the issue:

https://stackblitz.com/run?file=app/typeahead-basic.ts

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 10.2.3

ng-bootstrap: 7.0.0

Bootstrap: 4.3.1

**Comment by maxokorokov** [comment]
Indeed, I think the confusion was resolved with https://github.com/w3c/aria/issues/1176
